src/Photos.vue

```diff
@@ -102,7 +103,7 @@ export default {
 		}
 	},
 
-	beforeMount() {
+	async beforeMount() {
```

Comment:
I don't think vue supports async lifecycle hooks :thinking:

Reply:
AFAIK, Vue doesn't care because it doesn't use the return value. An async function is just syntactic sugar for a function that returns a promise, after all.